README.md

```diff
@@ -1,7 +1,7 @@
 # swag-for-dev
 
 [![Build Status](https://github.com/swapagarwal/swag-for-dev/workflows/All%20the%20CI/badge.svg)](https://github.com/swapagarwal/swag-for-dev/actions)
-[![Renovate](https://badges.renovateapi.com/github/swapagarwal/swag-for-dev)](https://renovatebot.com/)
+[![Renovate](https://img.shields.io/badge/Renovate-swag--for--dev-green)](https://renovatebot.com/)
```

Comment:
I prefer that one: https://img.shields.io/badge/renovate-enabled-green?logo=renovatebot
What do you think?